pages/classes/index.vue

```diff
@@ -13,7 +13,7 @@
       />
     </div>
     <api-table-filter
-      :update-filters="update"
+      :update-filters="updateFilter"
```

Comment:
Renamed for more clarity (we might have another `update` function that does something else)